api/src/features/posts/models/requests/PostUpdate.ts

```diff
@@ -0,0 +1,75 @@
+import { AttributeValue, AttributeValueUpdate } from '@aws-sdk/client-dynamodb';
```

Comment:
This is ugly. Good for now, but probably an area for improvement. I would extract UpdateExpressionBuilder though, since its reusable